src/Tools/dotnet-user-jwts/src/Helpers/ConsoleTable.cs

```diff
@@ -59,25 +57,94 @@ public void Write()
 
-        var formattedRows = _rows.Select(row => string.Format(CultureInfo.InvariantCulture, formatRow, row)).ToList();
         var columnHeaders = string.Format(CultureInfo.InvariantCulture, formatRow, _columns.ToArray());
         var rowDivider = $" {new string('-', columnHeaders.Length - 1)} ";
 
+        // Creates a nested list of items, each representing a single jwt
+
+        var formattedRows = new List<List<string>>();
+
+        foreach (var jwtObject in _rows)
+        {
+            var stringListOfJwtItems = new List<string>();
+            var jwtList = jwtObject.ToList();
+
+            for (var i = 0; i < jwtList.Count; i++)
+            {
+                stringListOfJwtItems.Add(jwtList[i].ToString());
+            }
+            formattedRows.Add(stringListOfJwtItems);
+        }
+
         builder.AppendLine(rowDivider);
         builder.AppendLine(columnHeaders);
 
-        foreach (var formattedRow in formattedRows)
+        var newFormattedRows = WriteTableContent(formattedRows, maxColumnLengths, rowDivider);
+
+        builder.AppendLine(rowDivider);
+
+        foreach (var formattedRow in newFormattedRows)
         {
-            builder.AppendLine(rowDivider);
             builder.AppendLine(formattedRow);
         }
 
-        builder.AppendLine(rowDivider);
-
         _reporter.Output(builder.ToString());
+
+        // Write each jwt into the table making sure that longer items are wrapped.
+        static List<string> WriteTableContent(List<List<string>> rows, List<int> columnLengths, string divider)
+        {
+            var listOfRows = new List<string>();
+            for (var i = 0; i < rows.Count; i++)
+            {
+                var updatedRow = rows[i];
+
+                bool status = true;
+                while (status)
+                {
+                    var outputRow = "";
+
+                    for (var j = 0; j < updatedRow.Count; j++)
+                    {
+                        outputRow = string.Concat(outputRow, " | ");
+                        var currentItem = updatedRow[j];
+
+                        if (currentItem.Length <= columnLengths[j])
+                        {
+                            outputRow = string.Concat(outputRow, currentItem, new string(' ', columnLengths[j] - currentItem.Length));
+                            updatedRow[j] = "";
+                        }
+                        else
+                        {
+                            outputRow = string.Concat(outputRow, currentItem.Substring(0, columnLengths[j]));
+                            updatedRow[j] = currentItem.Substring(columnLengths[j]);
+                        }
+                    }
+                    outputRow = string.Concat(outputRow, " |");
+                    listOfRows.Add(outputRow);
+
+                    // Check if all items in updated row are set to an empty string
+                    if (updatedRow.All(item => item == ""))
```

Comment:
nit: I would track if we ever went into the above `else` statement and if we didn't then break from the loop.